### PR TITLE
Support running pub when it's not on the environment path

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: webdev/lib/src/util.dart [hunk 1]
```diff
@@ -2,10 +2,26 @@
...
 import 'package:pub_semver/pub_semver.dart';
 
 final supportedBuildRunnerVersionRange = new VersionRange(
     min: new Version(0, 8, 0),
     includeMin: true,
     max: new Version(0, 9, 0),
     includeMax: false);
+
+/// The path to the root directory of the SDK.
+final String _sdkDir = (() {
```
Comment (jakemac53) [inline webdev/lib/src/util.dart]: should we be doing this instead of using cli_util?

**Comment by alexander-doroshko** [comment]
You need to replace 'pub' with pubPath in [pubspec.dart:41](https://github.com/dart-lang/webdev/pull/26/files#diff-ac21d7d2eb1b833007467f0a870fec1dR41) as well :)

File: webdev/lib/src/util.dart (hunk 1 above)
Comment (kevmoo) [reply webdev/lib/src/util.dart]: Not worth the dependency for a one-line method.

**Comment by alexander-doroshko** [comment]
Works fine for me now!

**Pull request merged**